### PR TITLE
[16.0] [ADD] partner_subject_to_vat

### PR DESCRIPTION
**Pull request by victor-champonnois**
New module adding a boolean fields to record whether the partner is subject to VAT. It does not affect the visibility of the VAT because the VAT information could be useful anyway.

### CODE REVIEW

File: partner_subject_to_vat/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    subject_to_vat = fields.Boolean(string="Subject to VAT")
```
Comment (carmenbianca) [inline partner_subject_to_vat/models/res_partner.py]:
```suggestion
    is_subject_to_vat = fields.Boolean(string="Is Subject to VAT")
```

**Review by robinkeunen — APPROVED** [review]

File: partner_subject_to_vat/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Partner Subject to VAT",
+    "summary": """
+        Record whether a company is subject to VAT.""",
+    "version": "16.0.1.0.0",
+    "category": "Contact",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "Coop IT Easy SC, Odoo Community Association (OCA)",
+    "maintainers": ["victor-champonnois"],
+    "license": "AGPL-3",
+    "application": False,
```
Comment (carmenbianca) [inline partner_subject_to_vat/__manifest__.py]: not needed

File: partner_subject_to_vat/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "depends": ["base"],
+    "excludes": [],
+    "data": ["views/res_partner_view.xml"],
+    "demo": [],
+    "qweb": [],
```
Comment (carmenbianca) [inline partner_subject_to_vat/__manifest__.py]: keys with empty values can be deleted

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Small suggestions, otherwise LGTM.

I leave it to the PSC to determine whether this module belongs in `partner-contact`

File: partner_subject_to_vat/__manifest__.py (hunk 3 above)
Comment (victor-champonnois) [reply partner_subject_to_vat/__manifest__.py]: fixed

File: partner_subject_to_vat/__manifest__.py (hunk 2 above)
Comment (victor-champonnois) [reply partner_subject_to_vat/__manifest__.py]: fixed

File: partner_subject_to_vat/models/res_partner.py (hunk 1 above)
Comment (victor-champonnois) [reply partner_subject_to_vat/models/res_partner.py]: fixed

**Review by carmenbianca — APPROVED** [review]

**Comment by victor-champonnois** [comment]
@gurneyalex @simahawk @bguillot Can this be merged ?

**Comment by carmenbianca** [comment]
@OCA/crm-sales-marketing-maintainers Kindly pinging for a merge/review :)

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

**Pull request merged**